### PR TITLE
Fixes for Pydantic 2.9 schema changes

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

**Comment by wRAR** [comment]
https://github.com/pydantic/pydantic/pull/10029

File: tests/test_params.py [hunk 1]
```diff
@@ -126,8 +126,8 @@ class ParamSpider(Args[Params], Spider):
                         "type": "boolean",
                     },
                     "fruit": {
-                        "allOf": [{"$ref": "#/$defs/FruitEnum"}],
-                        "title": "Fruit",
+                        "$ref": "#/$defs/FruitEnum",
```
Comment (kmike) [inline tests/test_params.py:129]: @wRAR just to make sure: does the SC UI support this format?

**Comment by kmike** [comment]
It looks unrelated, but we also should fix twinecheck (probably by adding license to package metadata?)

File: tests/test_params.py (hunk 1 above)
Comment (kmike) [reply tests/test_params.py:129]: Or is it the pydantic output, not the normalize schema?

Comment (wRAR) [reply tests/test_params.py:129]: This is raw output of the latest pydantic. The normalized schema stays the same.

**Comment by wRAR** [comment]
I was going to fix twinecheck either separately or when preparing the release.

**Review by kmike — APPROVED** [review]

**Pull request merged**